content/sensu-go/5.5/reference/backend.md

```diff
@@ -340,7 +340,7 @@ agent-port: 8081{{< /highlight >}}
 
 | cert-file  |      |
 -------------|------
-description  | SSL/TLS certificate
+description  | Specifies a fallback SSL/TLS certificate if the flag `dashboard-cert-file` is not use. This certificate secures communication with the Sensu Dashboard.
```

Comment:
Based on https://docs.sensu.io/sensu-go/5.5/guides/securing-sensu/#securing-the-api-and-the-dashboard, it seems like this flag is also used to secure the API.